hyph-utf8/tex/generic/hyph-utf8/patterns/tex/hyph-af.tex

```diff
@@ -1,30 +1,28 @@
-% Hyphenation patterns for Afrikaans
+% title: Hyphenation patterns for Afrikaans
+% copyright: Copyright (c) 2013 Tilla Fick (fickm@unisa.ac.za) and Chris Swanepoel (swanecj@unisa.ac.za)
+% notice: >
+%     This file is part of the hyph-utf8 package.
+%     See http://www.hyphenation.org for more information
+% language:
+%     name: Afrikaans
+%     tag: af
+% version: 1.0 (September 2013)
+% licence:
+%     name: LPPL
+%     version: 1.3
+%     or_later: true
 %
-% This work may be distributed and/or modified under the
-% conditions of the LaTeX Project Public License, either version 1.3
-% of this license or (at your option) any later version.
-% The latest version of this license is in
-%   http://www.latex-project.org/lppl.txt
-% and version 1.3 or later is part of all distributions of LaTeX
-% version 2005/12/01 or later.
-% -----------------------------------------------------------------
+% hyphenmins:
+%     generation:
+%         left: 1
+%         right: 1
+%     typesetting:
+%         left: 1
+%         right: 2
+% ==========================================
 % These patterns were generated with Opatgen from a lexicon
 % of 183 000 syllabified Afrikaans words.
-% -----------------------------------------------------------------
-% VERSION: 1.0 (September 2013)
 %
-% AUTHORS:  Tilla Fick (fickm@unisa.ac.za)
-%           and
-%           Chris Swanepoel (swanecj@unisa.ac.za)
-%
-% -----------------------------------------------------------------
-%
-% Define lefthyphenmin to one and righthyphenmin to two
-% (although righthyphenmin could technically be made one,
-%  in allmost all cases a value of two gives better results):
-% \lefthyphenmin=1 \righthyphenmin=2
```

Comment:
I find their comment valuable. The contents are included in yaml, but I would leave this paragraph somewhere.

Reply:
Reinstated.